cartridge_account/src/lib.cairo

```diff
@@ -214,18 +217,11 @@ mod Account {
             let tx_info = get_tx_info().unbox();
             let tx_hash = tx_info.transaction_hash;
             let mut signature = tx_info.signature;
-            if signature.len() == 2_u32 {
-                assert(self._is_valid_signature(tx_hash, signature), Errors::INVALID_SIGNATURE);
-                return starknet::VALIDATED;
-            } 
-
-            let signature_type = *signature.at(0);
-
-            if signature_type == starknet::VALIDATED {
-                assert(false, Errors::INVALID_SIGNATURE);
+            if self._is_valid_signature(tx_hash, signature) {
```

Comment:
```suggestion
            if self.is_valid_signature(tx_hash, signature) {
```
For consistency we may also removed the leading underscore since we're in a private impl already.

Reply:
Fixed in the subsequent PR. [feature/2/5](https://github.com/neotheprogramist/cairo-webauthn/pull/28)